cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/convertproto/CryptocurrencyConvertor.kt

```diff
@@ -0,0 +1,30 @@
+package com.ajax.cryptocurrency.application.convertproto
+
+import com.ajax.cryptocurrency.CryptocurrencyOuterClass.Cryptocurrency
+import com.ajax.cryptocurrency.application.util.toLocalDateTime
+import com.ajax.cryptocurrency.application.util.toTimestamp
+import com.ajax.cryptocurrency.domain.CryptocurrencyDomain
+import org.springframework.stereotype.Component
+
+@Component
+class CryptocurrencyConvertor {
+    fun cryptocurrencyToProto(
```

Comment:
protobuf is infrastructure component, it should not be used in application layer

Reply:
moved package to infrastructure layer